Project-Hotpot/Project-Hotpot/Errors.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  Errors.swift
+//  Project-Hotpot
+//
+//  Created by Harini Sundaram on 7/26/22.
+//
+
+import Foundation
+
+//MARK: Errors:
+public enum CustomError: Error {
```

Comment:
Nit: Similar to "ViewController", this name doesn't really convey much about what it's intended for. Perhaps something like "HotpotNetworkError"?